packages/malloy/src/malloy.ts

```diff
@@ -2258,7 +2246,7 @@ export class Runtime {
    */
   public loadModel(
     source: ModelURL | ModelString,
-    options?: ParseOptions & CompileOptions
+    options?: ParseOptions & CompileOptions & PrepareResultOptions
   ): ModelMaterializer {
```

Comment:
When you load a model, you might want to pass in `replaceMaterializedReferences` immediately because of SQL blocks. To make this less confusing, I've added `replaceMaterializedReferences` onto `ModelMaterializer` so that if you do:

```ts
const model = runtime.loadModel(..., { replaceMaterializedReferences: true });
const query = model.loadQuery(...);
const sql = query.getPreparedResult();
```
The `replaceMaterializedReferences` will get stored in the `ModelMaterializer` in the first call, then passed through to the `QueryMaterializer` in the second call, and then used by `getPreparedResult` when running the compiler.